tests/perf/linux_perf_event.cc

```diff
@@ -90,5 +90,6 @@ linux_perf_event::user_instructions_retired() {
             .disabled = 1,
             .exclude_kernel = 1,
             .exclude_hv = 1,
+            .exclude_idle = 1,
```

Comment:
I don't think this is meaningful.

```
       exclude_idle
              If set, don't count when the CPU is running the idle task.  While you can currently enable this for any event type, it is ignored for all but software events.
```

But, we're already tied to a thread, so we aren't executing the idle task.